src/sssom/parsers.py

```diff
@@ -193,44 +194,35 @@ def parse_sssom_table(
     stream: io.StringIO = _open_input(file_path)
     sep_new = _get_seperator_symbol_from_file_path(file_path)
     df, sssom_metadata = _read_pandas_and_metadata(stream, sep_new)
-    # if mapping_predicates:
-    #     # Filter rows based on presence of predicate_id list provided.
-    #     df = df[df["predicate_id"].isin(mapping_predicates)]
-
-    # If SSSOM external metadata is provided, merge it with the internal metadata
-
-    if sssom_metadata:
-        if meta:
-            for k, v in meta.items():
-                if k in sssom_metadata:
-                    if sssom_metadata[k] != v:
-                        logging.warning(
-                            f"SSSOM internal metadata {k} ({sssom_metadata[k]}) "
-                            f"conflicts with provided ({meta[k]})."
-                        )
-                else:
-                    logging.info(f"Externally provided metadata {k}:{v} is added to metadata set.")
-                    sssom_metadata[k] = v
-        meta = sssom_metadata
-
-        if PREFIX_MAP_KEY in sssom_metadata:
-            if prefix_map:
-                for k, v in prefix_map.items():
-                    if k in sssom_metadata[CURIE_MAP]:
-                        if sssom_metadata[CURIE_MAP][k] != v:
-                            logging.warning(
-                                f"SSSOM prefix map {k} ({sssom_metadata[CURIE_MAP][k]}) "
-                                f"conflicts with provided ({prefix_map[k]})."
```

Comment:
Is this warning preserved in curies.chain?

Reply:
No. The notion of logging conflicts here was more to support how complicated this code was. The `curies.chain` goes one step further than the previous code and creates a coherent, merged prefix map that loses no information from any of the inputs. Only in a case where there is an irreconcilable difference does `curies.chain` throw an error.

Reply:
So when the original sssom file provided map contains `a: http://a.b` and say the `prefix_map` parameter provided map contains `a: http://a.c` it will just maintain the on provided by the sssom file provided map. What happens if the `prefix_map` provided map contains `x: http://a.b`?

Reply:
> So when the original sssom file provided map contains a: http://a.b and say the prefix_map parameter provided map contains a: http://a.c it will just maintain the on provided by the sssom file provided map. 

This results in an EPM with:

```json
[ {"prefix": "a", "uri_prefix": "http://a.b", "uri_prefix_synonyms": ["http://a.c"]} ]
```

> What happens if the prefix_map provided map contains x: http://a.b?

You would get

```json
[ {"prefix": "a", "prefix_synonyms": ["x"], "uri_prefix": "http://a.b"} ]
```

Reply:
Ok cool, this makes sense. Thx.